src/test/java/seedu/eylah/expensesplitter/model/ReceiptStub.java

```diff
@@ -0,0 +1,24 @@
+package seedu.eylah.expensesplitter.model;
+
+import seedu.eylah.expensesplitter.model.receipt.Receipt;
+
+/**
+ * This simple ReceiptStub resembles Receipt in Expense Splitter but has limited functionality to ensure that
+ * that there is no error.
+ *
+ * Used in :
+ * PaidCommandTest
+ *
```

Comment:
I think this is a good way to let us keep track of the usage of this stub class.